### PR TITLE
fix: add artimanputra

### LINKED ISSUE #901: artimanputra
Screenshot_2

### PR DESCRIPTION
**Pull request by artimanputra**
Resolve #901 

## Checklist for profile addition

- [x] Name the PR `fix: Add <your-username>`.
- [x] You've added file named `your-username.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [x] You've linked the PR to the correct issue.

---

## Other additions/changes

- [ ] Changes made in this PR are mentioned below
- [ ] Search previous suggestions before making a new one, as yours may be a duplicate.
- [ ] I acknowledge that all my contributions will be made under the project's license.
- [ ] I've read the [contributing guidelines](https://github.com/EddieHubCommunity/awesome-github-profiles/blob/main/CONTRIBUTING.md)

### CODE REVIEW

File: profiles/artimanputra.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "image": "https://user-images.githubusercontent.com/79042027/145818832-0bdad31d-7dfb-423f-8c76-f51a14c202e1.png",
+    "issueId": 901,
+    "name": "Arti Manputra",
+    "username": "artimanputra"
+  }
```
Comment (krishguptadev) [inline profiles/artimanputra.json:6]:
```suggestion
}
```

File: profiles/artimanputra.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+    "image": "https://user-images.githubusercontent.com/79042027/145818832-0bdad31d-7dfb-423f-8c76-f51a14c202e1.png",
```
Comment (krishguptadev) [inline profiles/artimanputra.json:2]:
```suggestion
    "image": "https://user-images.githubusercontent.com/91655303/145820969-c180ba53-ed0d-4992-8bc2-f40c53b05f3b.png",
```

I recommend a pic like => 
image

File: profiles/artimanputra.json (hunk 1 above)
Comment (Panquesito7) [inline profiles/artimanputra.json:6]: Missing an endline.
```suggestion
}
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Please see https://github.com/EddieHubCommunity/awesome-github-profiles/pull/902#discussion_r767756290. The image suggested by @krishguptadev is better, IMO.

File: profiles/artimanputra.json (hunk 1 above)
Comment (krishguptadev) [inline profiles/artimanputra.json:6]:
```suggestion
}

```

**Comment by artimanputra** [comment]
> Please see [#902 (comment)](https://github.com/EddieHubCommunity/awesome-github-profiles/pull/902#discussion_r767756290). The image suggested by @krishguptadev is better, IMO.

Made the requested changes @Panquesito7 and @krishguptadev.

File: profiles/artimanputra.json [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{
+    "image": "https://user-images.githubusercontent.com/91655303/145820969-c180ba53-ed0d-4992-8bc2-f40c53b05f3b.png",
+    "issueId": 901,
+    "name": "Arti Manputra",
+    "username": "artimanputra"
+}
```
Comment (Panquesito7) [inline profiles/artimanputra.json:6]:
```suggestion
}

```

File: profiles/artimanputra.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
-    "image": "https://user-images.githubusercontent.com/79042027/145818832-0bdad31d-7dfb-423f-8c76-f51a14c202e1.png",
+    "image": "https://user-images.githubusercontent.com/91655303/145820969-c180ba53-ed0d-4992-8bc2-f40c53b05f3b.png",
     "issueId": 901,
     "name": "Arti Manputra",
     "username": "artimanputra"
-  }
\ No newline at end of file
+}
```
Comment (krishguptadev) [inline profiles/artimanputra.json]:
```suggestion
}

```

Comment (artimanputra) [reply profiles/artimanputra.json]: > 

Can you please tell me what I have to do like I am not getting anything?

Comment (krishguptadev) [reply profiles/artimanputra.json]: new line

File: profiles/artimanputra.json [hunk 5]
```diff
@@ -0,0 +1,7 @@
+{
+    "image": "https://user-images.githubusercontent.com/91655303/145820969-c180ba53-ed0d-4992-8bc2-f40c53b05f3b.png",
+    "issueId": 901,
+    "name": "Arti Manputra",
+    "username": "artimanputra"
+}
+
```
Comment (Panquesito7) [inline profiles/artimanputra.json:7]:
```suggestion
```

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

**Pull request merged**